qemu/tests/numa_cpu.py

```diff
@@ -63,6 +70,9 @@ def numa_cpu_guest():
                                          'iasl -d srat.dat && cat srat.dsl')
             pattern = re.compile(r'Proximity Domain Low\(8\)\s+:\s+([0-9A-Fa-f]+)'
                                  r'\n.*Apic ID\s+:\s+([0-9A-Fa-f]+)')
+            if vm_arch == 'aarch64':
+                pattern = re.compile(r'Proximity Domain\s+:\s+([0-9A-Fa-f]+)'
+                                     r'\n.*Acpi Processor UID\s+:\s+([0-9A-Fa-f]+)')
```

Comment:
The output format on ARM is hex, I am not sure if we need to handle something. Please correct me if I am wrong.

Reply:
don't worry, in the code below, there is a conversion hex to dec, so we don't need extra processing
```
                nodeid = int(item[0], 16)
                cpuid = int(item[1], 16)
```